topsort-api-v2.yml

```diff
@@ -134,7 +134,11 @@ paths:
                           winnerType: product
                           winnerId: p_PJbnN
                           resolvedBidId: WyJlX1BKYm5OIiwiMTJhNTU4MjgtOGVhZC00Mjk5LTgzMjctY2ViYjAwMmEwZmE4IiwiYmFubmVyQWRzIiwiZGVmYXVsdCIsIiJd
-                          assetUrl: https://topsort.cdnprovider.com/lhs-banner-image-for-p_PJbnN.png
+                          assets:
+                            - factor: 1x
+                              url: https://topsort.cdnprovider.com/lhs-banner-image-for-p_PJbnN-1x.png
```

Comment:
Is factor the correct word

Reply:
Dimensions sounds good. Since this is a breaking change and we haven't implemented this yet, I'll update this PR but not merge it to avoid confusing marketplaces currently testing with it (OR we can release this in v3 but then it would make little sense to support v2 as such a short lived API).

Reply:
We can always keep the old field nd just return the url corresponding to he highest resolution (original) of the asset.